lib/nationbuilder/client.rb

```diff
@@ -56,6 +56,9 @@ def raw_call(path, method, body = {}, args = {})
       request_args[:query].merge!(body)
     else
       body[:access_token] = @api_key
+      if !body[:fire_webhooks].nil?
+        request_args[:query][:fire_webhooks] = body[:fire_webhooks]
```

Comment:
Alas, I don't think this is quite what we want, as not setting `:fire_webhooks` in the `body` will mean that it gets set to `false`, which then means we're disabling webhooks by default.